package/piwebcam/multi-gadget.sh

```diff
@@ -1,30 +1,59 @@
 #!/bin/sh
+#
+# Thank to 
+#   wismna: http://github.com/wismna/raspberry-pi/hackpi
+#   RoganDawes: https://github.com/RoganDawes/P4wnP1
 
+# https://docs.microsoft.com/en-us/windows-hardware/drivers/usbcon/enumeration-of-the-composite-parent-device
+# https://docs.microsoft.com/en-us/windows-hardware/drivers/usbcon/usb-common-class-generic-parent-driver
+
+OS_CONFIG="$1"
+MAC_HOST="48:6f:73:74:50:43" # "Mac address for host PC"
+MAC_SELF="42:61:64:55:53:42" # "MAC address for PiCam"
 CONFIG=/sys/kernel/config/usb_gadget/piwebcam
+
 mkdir -p "$CONFIG"
 cd "$CONFIG" || exit 1
 
-echo 0x1d6b > idVendor
-echo 0x0104 > idProduct
+
+# 0x04b3 = 1203
+# {
+#     "name": "IBM Corporation",
+#     "field_vid": "1203"
+# },
+OS_CONFIG=$(echo $OS_CONFIG | tr '[A-Z]' '[a-z]')
+case $OS_CONFIG in
```

Comment:
Better way to do this is to have config file to see what usb gadget we want to configure, and the decision to ask what the pi wants to be is left to the user (we can suggest that if you use windows, then configure it as RNDIS, etc.)